### PR TITLE
X fields and Aliases

### PR DESCRIPTION
**Pull request by David-BMS**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/43577)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/IncidentFields/incidentfield-Agent_IDs.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        },
+        {
+          "cliName": "sensorname",
+          "type": "shortText"
+        },
+        {
+          "cliName": "trapsid",
+          "type": "shortText"
+        }
+      ]
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Agent_IDs.json:45]: Missing Agents_ID in the list

File: Packs/CommonTypes/IncidentFields/incidentfield-Appliance_IDs.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
+    "x2_fields": "fw_serial_number",
+    "Aliases": [
+        {
+          "cliName": "applianceid",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Appliance_IDs.json:34]: Add "marketplace xsoar" to the field

File: Packs/CommonTypes/IncidentFields/incidentfield-Appliance_Names.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
+    "x2_fields": "fw_device_name",
+    "Aliases": [
+        {
+          "cliName": "appliancename",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Appliance_Names.json:34]: Add "marketplaces xsoar" to the field @

File: Packs/CommonTypes/IncidentFields/incidentfield-Application_Sub_Category_Name.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
+
+{
+    "associatedToAll": true,
+    "caseInsensitive": true,
+    "cliName": "applicationcategoryname",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Application_Sub_Category_Name.json:5]: fix cliName, description, id

File: Packs/CommonTypes/IncidentFields/incidentfield-Country_Names.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
+	  "x2_fields": "action_country",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Country_Names.json:31]: fix indentation

File: Packs/CommonTypes/IncidentFields/incidentfield-Email_Subjects.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
+	"x2_fields": "fw_email_subject",
+    "Aliases": [
+        {
+          "cliName": "emailsubject",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Email_Subjects.json:34]: Add xsoar marketplace

File: Packs/CommonTypes/IncidentFields/incidentfield-Event_Type_Names.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
+	"x2_fields": "event_type",
+    "Aliases": [
+        {
+          "cliName": "eventtype",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Event_Type_Names.json:33]: Add xsoar marketplace to the field

File: Packs/CommonTypes/IncidentFields/incidentfield-Process_names.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+    "required": false,
+    "sla": 0,
+    "system": false,
+    "threshold": 72,
+    "type": "multiSelect",
+    "unmapped": false,
+    "unsearchable": true,
+    "useAsKpi": false,
+    "version": -1,
+    "fromVersion": "5.5.0",
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Process_names.json:30]: Add x2_fields

**Review by reutshal — CHANGES_REQUESTED** [review]

File: Packs/CommonTypes/IncidentFields/incidentfield-Agent_IDs.json [hunk 9]
```diff
@@ -38,6 +38,10 @@
           "cliName": "sensorname",
           "type": "shortText"
         },
+        {
+          "cliName": "trapsid",
+          "type": "shortText"
+        },
```
Comment (reutshal) [inline Packs/CommonTypes/IncidentFields/incidentfield-Agent_IDs.json:44]: remove

**Review by reutshal — APPROVED** [review]

**Comment by reutshal** [comment]
@David-BMS Can we close this?

**Comment by David-BMS** [comment]
@reutshal I'm using it for something else now
I'll close it soon

**Comment by David-BMS** [comment]
Closing due to multiple changes since

**Pull request closed without merging**